test/kinesis/wait-for-ready.test.js

```diff
@@ -0,0 +1,25 @@
+const test = require('ava');
+const sinon = require('sinon');
+
+const AWS = require('aws-sdk');
+const AWSMock = require('aws-sdk-mock');
+AWSMock.setSDKInstance(AWS);
+
+// Setup a mock listTables call that will fail once
+// to force a retry and then success to allow the setup
+// to continue
+const listStreamsMock = sinon.stub()
+  .resolves({ StreamNames: [] }, [])
+  .onFirstCall().rejects(new Error('First error'));
+
+AWSMock.mock('Kinesis', 'listStreams', listStreamsMock);
+
+const { useKinesisDocker } = require('../../src/kinesis');
+
+useKinesisDocker(test);
+
+test.serial('The helper provides database clients and tables', async (test) => {
```

Comment:
This test description does not look accurate.

Reply:
CPE